### PR TITLE
Licenses added

### PR DESCRIPTION
**Pull request by andriikaplanovskyi**
Package licenses added

### CODE REVIEW

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.AzureTableStorage/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+OneBeyond Studio entity audting implemented based on Azure table storage.
```
Comment (fabiomaistro) [inline src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.AzureTableStorage/readme.md:1]: Typo `auditing`

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Infrastructure/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+OneBeyond Studio entity audting infrastructure.
```
Comment (fabiomaistro) [inline src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Infrastructure/readme.md:1]: Typo `auditing`

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.SqlServer/readme.md [hunk 3]
```diff
@@ -0,0 +1 @@
+OneBeyond Studio entity audting implemented based on SQL Server.
```
Comment (fabiomaistro) [inline src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.SqlServer/readme.md:1]: Typo `auditing`

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Domain/readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+OneBeyond Studio core conepts and structures for entity auditing.
```
Comment (fabiomaistro) [inline src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Domain/readme.md:1]: Typo `concepts`

**Review by fabiomaistro — CHANGES_REQUESTED** [review]

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.AzureTableStorage/readme.md (hunk 1 above)
Comment (andriikaplanovskyi) [reply src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.AzureTableStorage/readme.md]: fixed

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.SqlServer/readme.md (hunk 3 above)
Comment (andriikaplanovskyi) [reply src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.SqlServer/readme.md]: fixed

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Domain/readme.md (hunk 4 above)
Comment (andriikaplanovskyi) [reply src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Domain/readme.md]: fixed

File: src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Infrastructure/readme.md (hunk 2 above)
Comment (andriikaplanovskyi) [reply src/OneBeyond.Studio.EntityAuditing/OneBeyond.Studio.EntityAuditing.Infrastructure/readme.md]: fixed

**Pull request merged**